File: python/prism_sdk/multimodal_audit_integrity_workflow_fabric.py
"""Sweep P32 multimodal workflow_fabric audit-integrity feature F14."""
from .audit_integrity_support import AuditRequest4,AuditCard7,AuditIntegrityError,manifest,qualify
FEATURE_ID="AFA-sweep-P32-F14";CONTRACT_VERSION="sweep-multimodal_audit_integrity_workflow_fabric/1.0"
def multimodal_audit_integrity_workflow_fabric_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="multimodal",mode="workflow_fabric")
def qualify_multimodal_audit_integrity_workflow_fabric(request:AuditRequest4)->AuditCard7:return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="multimodal",mode="workflow_fabric")

File: crates/sweep/src/multimodal_audit_integrity_workflow_fabric.rs
//! Sweep P32 multimodal workflow_fabric audit-integrity feature.
use super::audit_integrity_support::{manifest,qualify,AuditCard7,AuditIntegrityError,AuditRequest4};
pub const FEATURE_ID:&str="AFA-sweep-P32-F14";pub const CONTRACT_VERSION:&str="sweep-multimodal_audit_integrity_workflow_fabric/1.0";
pub fn multimodal_audit_integrity_workflow_fabric_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"multimodal","workflow_fabric")}
pub fn qualify_multimodal_audit_integrity_workflow_fabric(request:&AuditRequest4)->Result<AuditCard7,AuditIntegrityError>{qualify(request,FEATURE_ID,CONTRACT_VERSION,"multimodal","workflow_fabric")}
